jobs/atc/spec

```diff
@@ -636,6 +636,9 @@ properties:
   vault.url:
     description: |
       Vault server URL to use for parameterizing credentials.
+  vault.cache:
+      description: |
+        Enable Vault cache for secrets lease duration in memory.
```

Comment:
This needs a `default: false` otherwise the template will fail when the user hasn't specified it.